### PR TITLE
Support multiple tasks marking unmarking and deleting

### PR DESCRIPTION
**Pull request by rachx**
Now,
mark 1 2 3
mark 1,2, 3
mark 1-4 5-7
will work.

Won't update GUI tests here since other parts of the GUI tests needs to be fixed first.
- [x] Pass multiple tasks into model

### CODE REVIEW

**Comment by rachx** [comment]
Fixes #36

**Comment by INCENDE** [comment]
Invalid command format is when parsing of the command fails:
- `mark one two three`
- `mark the whole list`

Invalid index is when the format is correct, but index does not exist on the list:
- `mark 0`
- `mark -1` (or any negative numbers)
- `mark 100` (for example let's say you only have 10 tasks)

**Review by burnflare — APPROVED** [review]
Good!

File: src/main/java/seedu/agendum/logic/parser/Parser.java [hunk 1]
```diff
@@ -236,6 +237,37 @@ private Command prepareSelect(String args) {
...
+     */
+    private Set<Integer> parseIndexes(String args) {
+        final Matcher matcher = TASK_INDEXES_ARGS_FORMAT.matcher(args.trim());
+        Set<Integer> taskIds = new HashSet<Integer>();
+
+        if (!matcher.matches()) {
+            return taskIds;
+        }
+
+        args = args.replaceAll("[ ]+", ",").replaceAll(",+", ",");
```
Comment (burnflare) [inline src/main/java/seedu/agendum/logic/parser/Parser.java:235]: nice

**Review by burnflare — APPROVED** [review]
This PR is good. Merge?

**Pull request merged**